.github/workflows/chocolatey-test.yml

```diff
@@ -9,12 +9,31 @@ jobs:
     runs-on: windows-latest
     steps:
       - uses: actions/checkout@v4
+
+      - name: Extract version from nuspec
+        id: get_version
+        shell: pwsh
+        run: |
+          $nuspecContent = Get-Content -Path chocolatey/stax/stax.nuspec -Raw
```

Comment:
Let's put those two steps into powershell script in repository, so we can reuse it in `.github/workflows/chocolatey-test.yml` and  `.github/workflows/chocolatey-release.yml`